v2.1/admin-ui-statements-page.md

```diff
@@ -0,0 +1,110 @@
+
+To view the **Statements** page, open [http://localhost:8080/#/statements](http://localhost:8080/#/statements) in your browser (replacing `localhost` and `8080` with your node's host and port).
+
+<div id="toc"></div>
+
+{{site.data.alerts.callout_danger}}
+**This feature is a work in progress**. It will change leading up to the v2.1 release.
+{{site.data.alerts.end}}
+
+## Limitations
+
+- If you have multiple applications running on the cluster, the **Statements** page shows cumulative parameter values across all applications, while the **Statements Details** page shows the values for the first application only.
+- The **Statements** page provides the SQL statement details only for the [gateway node](architecture/sql-layer.html#overview). To view the details for other nodes, [access the Admin UI](admin-ui-access-and-navigate.html#access-the-admin-ui) from that node and navigate to `http://<node address>:8080/#/statements` from the browser.
+- The **Statements** page displays the details of the SQL statements executed only within a specified time interval. By default, the time interval is set to one hour; however, you can customize the interval using the [`diagnostics.reporting.interval`](cluster-settings.html#settings) cluster setting.
+
+## Understanding the Statements page
+
+### SQL statement fingerprint
+
+Whenever possible, the **Statements** page displays the details of SQL statement fingerprints instead of individual SQL statements.
+
+A statement fingerprint is a grouping of similar SQL statements in their abstracted form by replacing the parameter values with `_`. Grouping similar SQL statements as fingerprints helps you quickly identify the frequently executed SQL statements and their latencies.
+
+A statement fingerprint is generated when two or more statements have the same abstractions. For example, the following statements have the same abstractions:
+
+`INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES (380, 11, 11098)`
+`INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES (192, 891, 20)`
+`INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES (784, 452, 78)`
+
+Thus they can have the same fingerprint:
+
+`INSERT INTO new_order(product_id, customer_id, no_w_id) VALUES (_, _, _)`
+
+The following statements are different enough to not have the same fingerprint:
+
+`INSERT INTO orders(product_id, customer_id, transaction_id) VALUES (380, 11, 11098)`
+`INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES (380, 11, 11098)`
+`INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES ($1, 11, 11098)`
+`INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES ($1, $2, 11098)`
+`INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES ($1, $2, $3)`
```

Comment:
I've opened cockroachdb/cockroach#27052 to track the idea to use placeholders for fingerprints.  Pending the way that gets resolved these examples may move up to the block above!